labrad/test/test_decorators.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+
+from labrad.decorators import setting
+from labrad.server import LabradServer
+from labrad import types
+
+
+class DummyServer(LabradServer):
+    @setting(100, param0='s: this is a string',
+                  param1='b: this is a boolean')
+    def dummy_setting(self, c, param0, param1):
+        pass
+
+
+def test_setting_types_with_comments():
+    accepted_tags = DummyServer.dummy_setting.accepts
+    print accepted_tags
```

Comment:
LOL, will remove, that was just for my own info while testing the test. FYI, you can see stdout by including the [`-s` option](https://pytest.org/latest/capture.html) when running pytest (not that we should get in this habit, but it's useful on occasion when debugging a test).